Python/pywarpx/Particles.py

```diff
@@ -12,17 +12,17 @@
 electrons = Bucket('electrons')
 electrons.charge = "-q_e"
 electrons.mass = "m_e"
-electrons.injection_style = "python"
+electrons.injection_style = "none"
```

Comment:
What actually happens if someone puts the pythonic
```suggestion
electrons.injection_style = None
```
here? :)
Can we map that to "none", too? :)

Reply:
Setting `electrons.injection_style = None` will work if `injection_style` has a default in c++. When a variable is set to `None`, it is not included as in input parameter. I think setting it to `None` is a good idea so that the default value is specified in only one place, in the c++.

Reply:
Thank you both. Done in https://github.com/ECP-WarpX/WarpX/pull/2081/commits/9066432544aac65d27504a6639e1388b3d8c9314.